app-near/scripts/seeds/su.ts

```diff
@@ -1,32 +1,88 @@
 import { faker } from "@faker-js/faker";
+import {
+  AgeCategory,
+  Gender,
+  ProfessionalCategory,
+  type Quartier,
+} from "@prisma/client";
 import { db } from "~/server/db";
 import { buildSuAnswer } from "~/server/test-utils/create-data/suAnswer";
+import { type CategoryStat, categoryStatQuartierMap } from "~/types/SuAnswer";
+import { select } from "weighted";
 
 export enum SurveyCase {
-  LESS_THAN_TARGET = "LESS_THAN_TARGET",
-  MORE_THAN_TARGET = "MORE_THAN_TARGET",
+  LESS_THAN_GLOBAL_TARGET = "LESS_THAN_GLOBAL_TARGET",
+  MORE_THAN_GLOBAL_TARGET = "MORE_THAN_GLOBAL_TARGET",
+  MORE_THAN_CATEGORIES_TARGETS = "MORE_THAN_CATEGORIES_TARGETS",
 }
 
+const getAnswerTargetsByCategories = async (
+  surveyId: number,
+): Promise<Record<CategoryStat, number>> => {
+  const neighborhood = await db.quartier.findFirst({ where: { surveyId } });
+  if (!neighborhood) {
+    const existingSurveys = await db.survey.findMany({
+      select: { name: true },
+    });
+
+    throw new Error(`
+survey not found. 
+
+Usage: npm run seed -- scope=su_answer surveyName=14e_arr surveyTarget=60 surveyCase=LESS_THAN_TARGET
```

Comment:
:green_circle: 
```suggestion
Usage: npm run seed -- scope=su_answer surveyName="Porte d'Orléans" surveyTarget=60 surveyCase=LESS_THAN_GLOBAL_TARGET
```

(à appliquer partout)